src/screens/DiscoveryScreenV2/fragment/TopicFragment.js

```diff
@@ -266,11 +266,11 @@ const TopicFragment = ({
 const styles = StyleSheet.create({
   fragmentContainer: {
     flex: 1,
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
   },
   noDataFoundContainer: {
     flex: 1,
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the background color of `fragmentContainer` and `noDataFoundContainer` is a significant visual change. Ensure that all text and other components within these containers have been updated with colors that provide sufficient contrast for readability.

```diff
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->